packages/frontend/src/components/message/Message.tsx

```diff
@@ -301,6 +301,12 @@ function buildContextMenu(
       label: tx('react'),
       action: handleReactClick,
     },
+    // Save Message
+    {
```

Comment:
shouldn't this be conditional, like show some un-save action if it is already saved?

Reply:
We could skip it, if it is already saved. But atm we have no "unsave" action, except deleting the saved message

Reply:
dc iOS has save and un-save

Reply:
@r10s how does un-save work on iOS?

Reply:
> @r10s how does un-save work on iOS?

by deleting the message-id returned by [`get_saved_msg_id()`](https://c.delta.chat/classdc__msg__t.html#ab13b251a52cf6b36627dff46f3c0be3e)

Reply:
Hmm, is that neccessary? I would say its sufficient to delete the saved message